webapp/templatetags/utils.py

```diff
@@ -109,7 +109,11 @@ def render(self, context):
                 extra_context[key] = value[1:-1]
             else:
                 # It's a variable
-                extra_context[key] = template.Variable(value).resolve(context)
+                try:
+                    variable = template.Variable(value)
```

Comment:
Why create this `variable` variable?

Reply:
Oh 'cos the line is too long?

Reply:
I would have done:

``` python3
extra_context[key] = template.Variable(value).resolve(
    context
)
```

Or let Black do it.

Reply:
To keep the linter happy, it flagged up the line length when using `extra_context[key] = template.Variable(value).resolve(context)` - do you see a better way?